MimeKit/Utils/ParseUtils.cs

```diff
@@ -390,7 +390,7 @@ public static bool TryParseDomain (byte[] text, ref int index, int endIndex, Rea
 			return TryParseDotAtom (text, ref index, endIndex, sentinels, throwOnError, "domain", out domain);
 		}
 
-		static ReadOnlySpan<byte> GreaterThanOrAt => new[] { (byte) '>', (byte) '@' };
+		static ReadOnlySpan<byte> GreaterThanOrAt => [(byte) '>', (byte) '@'];
 
```

Comment:
This could also potentially use the `static readonly ReadOnlySpan<byte> GreaterThanOrAt => ">@";` syntax.

Reply:
I was on the fence on whether converting to a string also lost some meaning, but agree this can be overlooked when the variable is well named. If we make this change, we still need the u8 suffix.